app/src/main/java/com/smashingboxes/epa_prototype_android/MainActivity.java

```diff
@@ -256,11 +282,12 @@ private void getAirQualityData() {
     }
 
     private void onAirQualityReceived(ArrayList<AirQuality> airQualities) {
+        airQualities.clear();
```

Comment:
I'm assuming you're clearing the data received from the response because you want to use the fake data for demo purposes?